substrate/frame/staking/src/pallet/mod.rs

```diff
@@ -2291,6 +2321,66 @@ pub mod pallet {
 			);
 			Ok(())
 		}
+
+		/// Chill an inactive validator.
+		///
+		/// - Anyone can call this function.
+		/// - The validator's stash account should be provided.
+		/// - The proof should be a list of consecutive sorted era indexes where the validator has
+		///   not produced any blocks.
+		/// - This will only be successful if the validator has not produced any blocks in
+		///   [`ChillInactiveValidatorThreshold`] consecutive eras.
+		#[pallet::call_index(30)]
+		#[pallet::weight(T::WeightInfo::chill_inactive_validator(proof.len() as _))]
+		pub fn chill_inactive_validator(
+			origin: OriginFor<T>,
+			stash: T::AccountId,
+			proof: BoundedVec<EraIndex, T::HistoryDepth>,
+		) -> DispatchResult {
+			ensure_signed(origin)?;
+			ensure!(Validators::<T>::contains_key(&stash), Error::<T>::NotValidator);
+
+			let threshold = ChillInactiveValidatorThreshold::<T>::get();
```

Comment:
```suggestion
			let threshold = ChillInactiveValidatorThreshold::<T>::get();
			
			if threshold.is_zero() { return Ok(()); }
```

Reply:
The threshold is not allowed to be zero, but it’s better to be safer.